schema_utils.py

```diff
@@ -627,3 +627,18 @@ def is_valid_audio_language_code(obj):
             bool. Whether the given object is a valid audio language code.
         """
         return utils.is_valid_language_code(obj)
+
+    @staticmethod
+    def is_string_contained_within_parenthesis(obj):
+        # type: (Text) -> bool
+        """Checks if the given obj (a string) is contained within parenthesis.
```

Comment:
Please do mention that the quotes is also necessary.

Reply:
Done